### DISCUSSION TITLE
Difficulty accessing dandisets in conda environment

### DISCUSSION THREAD

**Discussion by arubiano98** [dandi/helpdesk#45] [Q&A] [answered]
Hello! I need to access a couple of dandisets(.nwb files), and I'm currently using the Spyder GUI (Python 3.8, conda), so I chose to use `conda install -c conda-forge dandi` for the most part, rather than the `pip install` alternative. With this in mind, I can't seem to download my target files, even when I _do_ use `pip install dandi`. 

Essentially, if I use `dandi download <dandiset HTML>` , an `invalid syntax` error directed at the word _download_ is thrown; on the other hand, if I try to call the _download_ function in a different way, such as `dandi.download(<dandiset HTML>)`, then a `'module' object is not callable` error is thrown. 

Again, this happens regardless of which command I use to install the dandi library in the first place. I have also tried to use different methods from the NWB website without much success, but perhaps that's simply because I don't fully understand the documentation. Any help would be much appreciated!

**Comment by jwodder** [comment, marked as answer]
The "invalid syntax" error suggests that you're trying to run `dandi download` from within a Python REPL.  That's not right; you need to run it directly from a command-line shell.

**Reply by bendichter** [reply]
@arubiano98 

Yes, `dandi download <dandiset HTML>` should be run from a command-line shell such as Terminal. You might be able to get away with: 

`!dandi download <dandiset HTML>` from within your Spyder Python session window. I don't know spyder, but this is generally the command to run a command-line command directly from an IPython session, so I'm guessing it will work in Spyder.

**Reply by arubiano98** [reply]
Thank you both, that was definitely the issue and I got around it by using `!dandi download <dandiset HTML>` as @bendichter suggested! Now I just have to figure out how to access the .nwb file that was downloaded from the dandiset. Thanks again!
